src/Core/Grid/Definition/Factory/CmsPageDefinitionFactory.php

```diff
@@ -176,6 +188,21 @@ protected function getColumns()
                         'clickable_row' => true,
                     ])
                     )
+                    ->add((new SubmitRowAction('duplicate'))
+                    ->setName($this->trans('Duplicate', [], 'Admin.Actions'))
+                    ->setIcon('content_copy')
+                    ->setOptions([
+                        'method' => 'POST',
+                        'route' => 'admin_cms_pages_duplicate',
+                        'route_param_name' => 'cmsPageId',
+                        'route_param_field' => 'id_cms',
+                        'modal_options' => new ModalOptions([
+                            'title' => $this->trans('Duplicate page', [], 'Admin.Actions'),
```

Comment:
This wording is new. Do we really need to create a new string? Isn't "duplicate" enough? 

@l-delin